### PR TITLE
CTCTRADERS-1472: Updated nav following safety and security journey being implemented

### PR DESCRIPTION
**Pull request by kevincoatsworth**
…ing implemented.

### CODE REVIEW

File: app/navigation/SecurityDetailsNavigator.scala [hunk 1]
```diff
@@ -29,28 +30,39 @@ class SecurityDetailsNavigator @Inject()() extends Navigator {
...
 
    private def addDangerousGoodsCodeRoute(ua: UserAnswers, index: Index, mode: Mode) =
     (ua.get(AddDangerousGoodsCodePage(index)), ua.get(DangerousGoodsCodePage(index)), mode) match {
       case (Some(true), _, NormalMode) => Some(routes.DangerousGoodsCodeController.onPageLoad(ua.id, index, NormalMode))
-      case (Some(false), _, NormalMode) =>
-        Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index)) //todo -update when Security Trader Details section done
+      case (Some(false), _, NormalMode) => Some(dangerousGoodsCodeRoute(ua, index))
       case (Some(true), None, CheckMode)    => Some(routes.DangerousGoodsCodeController.onPageLoad(ua.id, index, CheckMode))
       case (Some(true), Some(_), CheckMode) => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
       case (Some(false), _, CheckMode)      => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
```
Comment (ashwinirudrappa) [inline app/navigation/SecurityDetailsNavigator.scala:52]:
```suggestion
      case (Some(_),_, CheckMode) => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
 ```
A minor observation, Can we combine the 2 conditions to one as both point to CYA

File: test/navigation/SecurityDetailsNavigatorSpec.scala [hunk 2]
```diff
@@ -76,27 +100,33 @@ class SecurityDetailsNavigatorSpec extends SpecBase with ScalaCheckPropertyCheck
...
       }
     }
-
   }
   "In CheckMode" - {
 
     "Must go from TransportChargesPage to AddItemsCheckYourAnswersPage" in {
       forAll(arbitrary[UserAnswers]) {
         answers =>
+          val updatedAnswers = answers
```
Comment (ashwinirudrappa) [inline test/navigation/SecurityDetailsNavigatorSpec.scala:124]: @kevincoatsworth do we need the val updatedAnswers, we can use answers directly

**Review by ashwinirudrappa — APPROVED** [review]
Looks good, Minor suggestions

File: app/navigation/SecurityDetailsNavigator.scala (hunk 1 above)
Comment (kevincoatsworth) [reply app/navigation/SecurityDetailsNavigator.scala]: :+1:

File: test/navigation/SecurityDetailsNavigatorSpec.scala (hunk 2 above)
Comment (kevincoatsworth) [reply test/navigation/SecurityDetailsNavigatorSpec.scala]: :+1:

**Pull request merged**